### ISSUE TITLE
Export complete info of gene to excel

### ISSUE THREAD

**Issue by LiThBio** [opengenomebrowser/opengenomebrowser#22]
Add a button to export the complete info of a gene (e.g. KEGG gene, KEGG reaction, Enogg, etc) to an excel file with options to select several or all columns and features
![image](https://user-images.githubusercontent.com/106086464/169828703-50d0a639-41ca-46d3-adb4-460ae2a7c426.png)
 of interest.

**Comment by MrTomRod** [comment]
I think the solution here could be to write a table view like [/genomes](https://opengenomebrowser.bioinformatics.unibe.ch/genomes/), only for /genes. Would that solve the issue?

**Comment by LiThBio** [comment]
Exactly, probably with an additional option to export to excel / csv / txt file to merge information of several genes from different organisms.

Thank you!
